sechub-cli/src/mercedes-benz.com/sechub/util/zip_test.go

```diff
@@ -139,7 +138,7 @@ func TestZipFileCanBeCreated_with_exclude_patterns_applied(t *testing.T) {
 		ZipWriter:          zipWriter,
 		PrefixInZip:        "",
 		Folders:            []string{dirname1, dirname2},
-		Excludes:           []string{"**/file3.txt", "**/f*0*.txt"},
+		Excludes:           []string{"**/file3.txt", "**/f*0.txt"},
```

Comment:
What is that change about?

Reply:
For whatever reason on my Mac and on CI/CD this exclude cause all (?) files to be excluded from the zip. I'll look into it further tomorrow